### PR TITLE
Some more text tweaks and fixes

### PR DESCRIPTION
**Pull request by AnnoyingGuide**
also added that warning to debug mode too.

### CODE REVIEW

File: Companions/Bree/BreeDialogue.cs [hunk 1]
```diff
@@ -496,7 +496,7 @@ public override string ControlMessage(Companion companion, ControlContext contex
                     if(PlayerMod.PlayerHasCompanionSummoned(MainMod.GetLocalPlayer, CompanionDB.Sardine))
                     {
                         if (Main.rand.Next(3) == 0)
-                            return "Don't think of trying something with my husband while in my body. Not only I'm watching, but he also knows it's you.";
+                            return "Don't think of trying anything with my husband while in my body. Not only I'm I watching you, but he also knows it's you.";
```
Comment (nakano15) [inline Companions/Bree/BreeDialogue.cs:499]: ???

Comment (nakano15) [inline Companions/Bree/BreeDialogue.cs]: Need to check this typo later...

File: Companions/Brutus/BrutusDialogue.cs [hunk 2]
```diff
@@ -84,23 +84,23 @@ public override string NormalMessages(Companion companion)
...
                     Mes.Add("*I'm not a fool, I'm certain that what [gn:1] does to [gn:2] is just to bully him. I hope he has an iron will, or else...*");
                     Mes.Add("*Do you think I should extend my bodyguard job to [gn:2] too? I don't like what [gn:1] does to him. But I don't want her to think badly of me, that could hurt my chan- I mean, my professionalism. What should I do?*");
-                    Mes.Add("*Earlier this day, [gn:2] tried to hire me to give a lesson to [gn:1]. I refused it for... Some reason.*");
+                    Mes.Add("*Earlier this day, [gn:2] tried to hire me to teach a lesson to [gn:1]. I refused it for... Some reason.*");
                 }
             }
             if (CanTalkAboutCompanion(3))
             {
-                Mes.Add("*You got a zombie on your town? How were you able to do that? I mean... Wow!*");
+                Mes.Add("*You got a zombie on your team? How were you able to do that? I mean... Wow!*");
```
Comment (nakano15) [inline Companions/Brutus/BrutusDialogue.cs]: Was actually meant to be town..

File: Companions/Celeste/CelestePreRecruitBehavior.cs [hunk 3]
```diff
@@ -119,7 +119,7 @@ private void OnAskBlessWorld()
             AskedBlessWorld = true;
             MultiStepDialogue m = new MultiStepDialogue();
             m.AddDialogueStep("*Yes. I prayed for "+MainMod.TgGodName+" to spread his blessing upon the world and its inhabitants.*");
-            m.AddDialogueStep("*That also means his blessing went to the Terrarians too, for treating well the TerraGuardians living here.*");
+            m.AddDialogueStep("*That also means his blessing went to the Terrarians too, for treating the TerraGuardians well that live here.*");
```
Comment (nakano15) [inline Companions/Celeste/CelestePreRecruitBehavior.cs:122]: ???

File: Companions/Brutus/BrutusDialogue.cs (hunk 2 above)
Comment (AnnoyingGuide) [reply Companions/Brutus/BrutusDialogue.cs]: Sorry I should have guessed that from the rest: would work as: in your town

File: Companions/Bree/BreeDialogue.cs (hunk 1 above)
Comment (AnnoyingGuide) [reply Companions/Bree/BreeDialogue.cs]: whoops missed that

File: Companions/Luna/TutoringDialogues.cs [hunk 4]
```diff
@@ -104,7 +104,7 @@ private static void AboutLeadingTerraGuardians()
             {
                 "*Depending on how much of a friend you are with someone, they may agree to follow you on your adventures. Some of them may not even mind joining your adventures at all, regardless of whether they know you or not.*",
                 "*You can also give them orders during your adventures, so they can take specific actions that you may find the need.*",
-                "*Pay attention to your group size. A companion may feel uncomfortable joining you if there are too many people in your group.*"
+                "*Pay attention to your group size. A companion may feel uncomfortable joining you if there are too many people in your group. Currently, you can 3 members in your group at once.*"
```
Comment (nakano15) [inline Companions/Luna/TutoringDialogues.cs:107]: I will need to change the number, to instead take the actual number of companions you can have following, so there is no need to keep updating this once the cap is increased.

Comment (AnnoyingGuide) [reply Companions/Luna/TutoringDialogues.cs:107]: Good Idea, not sure how to implement that one myself though

File: Companions/Michelle/MichelleDialogues.cs [hunk 5]
```diff
@@ -72,13 +72,13 @@ public override string NormalMessages(Companion guardian)
...
                 {
-                    Mes.Add("I was kicked out of [nn:" + Terraria.ID.NPCID.Mechanic + "]'s room... She didn't like when I turned off the light switches after she turned them on...");
+                    Mes.Add("I was kicked out of [nn:" + Terraria.ID.NPCID.Mechanic + "]'s room... She didn't like it when I turned off the light switches after she turned them on...");
                 }
 
                 if (CanTalkAboutCompanion(CompanionDB.Rococo))
                 {
                     Mes.Add("I like [gn:" + CompanionDB.Rococo + "] because he's so easy to befriend. Soon We'll be BFFs.");
-                    Mes.Add("[gn:" + CompanionDB.Rococo + "] took me outside someday to watch a meteor shower, at the top of a mountain. It was so beautiful...");
+                    Mes.Add("somedays [gn:" + CompanionDB.Rococo + "] takes me outside to watch a meteor shower, at the top of a mountain. It was so beautiful...");
```
Comment (nakano15) [inline Companions/Michelle/MichelleDialogues.cs]: Need review..

File: NpcMod.cs [hunk 6]
```diff
@@ -277,14 +277,14 @@ public override void GetChat(NPC npc, ref string chat)
...
+                        PossibleMessages.Add(WorldMod.GetCompanionNpcName(0) + " asked me how to use a blowpipe, then I tried teaching him. He nearly died of suffocation because he inhaled the seed instead.");
                     if (CanTalkAboutCompanion(1))
                         PossibleMessages.Add(WorldMod.GetCompanionNpcName(1) + " seems to have some interest in poisons, just watch out if she gives you a drink. Who knows?");
                     if (CanTalkAboutCompanion(8))
                         PossibleMessages.Add("Red paint? I didn't painted my mask red... Uh oh... Forget what you saw.");
                     break;
                 case Terraria.ID.NPCID.DD2Bartender:
-                    PossibleMessages.Add("Bringing a TerraGuardian to the defense of the crystal is a bit of a cheat, but It is very helpful when defending it alone.");
+                    PossibleMessages.Add("Bringing a TerraGuardian to the defense of the crystal kind of cheaty, but it is very helpful when defending it alone.");
```
Comment (nakano15) [inline NpcMod.cs]: is is missing.

Comment (AnnoyingGuide) [reply NpcMod.cs]: Fixed

File: Companions/Rococo/RococoDialogues.cs [hunk 7]
```diff
@@ -14,7 +14,7 @@ public override string GreetMessages(Companion companion)
             switch (Main.rand.Next(4))
             {
                 case 0:
-                    return "\"At first, the creature got surprised after seeing me, then starts laughing out of happiness.\"";
+                    return "\"At first, the creature got surprised after seeing me, then starts laughing out do to happiness.\"";
```
Comment (nakano15) [inline Companions/Rococo/RococoDialogues.cs]: ???

Comment (AnnoyingGuide) [reply Companions/Rococo/RococoDialogues.cs]: Uh, now I don't know what I did there at all lol. Let me fix that

Comment (AnnoyingGuide) [reply Companions/Rococo/RococoDialogues.cs]: Fixed

**Pull request merged**